src/pages/home/components/Filter/Filter.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import Slider from '../../../../shared/components/Slider/Slider';
+import './styles/Filter.scss';
+
+export default class Filter extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      abvSlider: '',
+      ibuSlider: '',
+      ebcSlider: '',
+    };
+  }
+
+  componentDidUpdate() {
+    const { doFilterBeers } = this.props;
+    const { ibuSlider: ibu, abvSlider: abv, ebcSlider: ebc } = this.state;
+
+    doFilterBeers({ ibu, abv, ebc });
+  }
+
+  onSliderChange = (e) => {
+    const { id, value } = e.target;
+
+    this.setState({ [id]: value });
+  };
+
+  render() {
+    const { ibuSlider, abvSlider, ebcSlider } = this.state;
+    const { isHidden } = this.props;
+    const classes = classNames('filter', { filter_hidden: isHidden });
+
+    return (
+      <div className={classes}>
+        <h3 className="filter__title">Filter results</h3>
+        <div className="filter__category">
+          Alcohol by volume
+          <Slider
+            id="abvSlider"
+            min="2"
```

Comment:
These random number values are called `magic numbers`. It's bad to have those, since it's hard to maintain.
Please move numbers to constants